scripts/apps/monitoring/aggregate-widget/aggregate.js

```diff
@@ -1,7 +1,7 @@
 angular.module('superdesk.apps.aggregate.widgets', ['superdesk.apps.aggregate', 'superdesk.apps.dashboard.widgets'])
     .config(['dashboardWidgetsProvider', function(dashboardWidgets) {
         dashboardWidgets.addWidget('aggregate', {
-            label: 'Monitoring',
+            label: 'Monitor',
```

Comment:
I think it should use gettext, maybe not for description but at least for this label